storm-core/src/jvm/backtype/storm/utils/ShellProcess.java

```diff
@@ -18,21 +18,17 @@
 package backtype.storm.utils;
 
 import backtype.storm.Config;
-import backtype.storm.multilang.ISerializer;
-import backtype.storm.multilang.BoltMsg;
-import backtype.storm.multilang.NoOutputException;
-import backtype.storm.multilang.ShellMsg;
-import backtype.storm.multilang.SpoutMsg;
+import backtype.storm.multilang.*;
```

Comment:
don't change imports if you dont have to. Different devs are using different IDEs